gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractJPackageTask.kt

```diff
@@ -231,6 +231,8 @@ abstract class AbstractJPackageTask @Inject constructor(
     @get:Optional
     val javaRuntimePropertiesFile: RegularFileProperty = objects.fileProperty()
 
+    private lateinit var jvmRuntimeInfo: JvmRuntimeProperties
```

Comment:
As I understand, it uses only once in if condition.
Maybe better to do it lazy?
val jvmRuntimeInfo by lazy{ ...